README.md

```diff
@@ -1 +1,44 @@
-# server-deployment-practice
\ No newline at end of file
+
+# LAB - Class 01b
+
+## Project: server-deployment-practice
+
+### Author: Student/Group Name
+
+Tina Myers
+
+### Links and Resources
+
+[GitHub](https://github.com/myerstina515/server-deployment-practice/tree/main)
+[Heroku Deployment](https://dashboard.heroku.com/apps/tinamyers-server-deploy-prod) 
```

Comment:
I would like to see the link to the actual app and not the dashboard for the application